### ISSUE TITLE
[CoE Starter Kit - QUESTION] Trigger is invalid for all 5 BYODL flows

### ISSUE THREAD

**Issue by anttipajunen** [microsoft/coe-starter-kit#5161]
### What is your question?

I'm getting a "the trigger is invalid" for all 5 BYODL flows and thus can't get them to fire off. Any ideas?

![image](https://user-images.githubusercontent.com/64043240/229227379-78b67764-1537-4579-a799-4deb482689ed.png)

![image](https://user-images.githubusercontent.com/64043240/229228088-086be165-e0c1-4505-86a3-8aeb288198ca.png)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.3.50

**Comment by manuelap-msft** [comment]
Hello,

looks like your Environment ID is incorrectly set - for Dataflows, the Environment contains a suffix that I don't see in your screenshot. We have a debugging flow for this in place, could you try the following steps

**Remove the current value for the CurrentEnvironment variable**
1. navigate to make.powerapps.com > Solutions
2. Select the Default Solution
3. Find the CurrentEnvironment environment variable
4. Remove the current value 

**Reset the CurrentEnvironment variable**
1. navigate to make.powerapps.com > Solutions
2. Open the CoE Core Components solution
3. Edit the BYODL>ConfigureEnvironment flow 
4. Manually select your current environment and the CoE BYODL Makers dataflow from the options (you might have to toggle Group type between Environment/Workspace/back to Environment to get the Group list to refresh and show all your environments)
![image](https://user-images.githubusercontent.com/55705669/229274830-19745be7-38c1-4390-8fab-56d07b22eec8.png)
5. Save and Test the flow
6. Once the flow has run, can you share back with us what the CurrentEnvironment value is. It should be your environment ID with a suffix - we are trying to set this env var as part of the setup wizard to include the suffix and tested this in most regions but I see your env is in Canada so I'll re-test it in that region in case we missed something there.

**Comment by anttipajunen** [comment]
Thanks @manuelap-msft.

TL;DR: Most likely PEBKAC, and I've set the env var based on the env value from PPAC. The 5 flows now fire off and refresh the dataflows.

Full version:

I'm fairly positive I've set the CurrentEnvironment value at solution import (new install). The env var is asking for an env ID, so I've just copied it from PPAC and slapped it in. This is most likely the cause of the error. Installing the preview for the first time, I've been jumping between the setup wizard, env vars, and dataflow, so I've been "all over" trying to pick the current content and "the flow of the installation" from Learn docs. I have to admit I did have an "I have no idea what I'm doing" feeling and had to kinda find my way through everything. My brain is constantly going back to how the non-preview version is installed and updated.

![image](https://user-images.githubusercontent.com/64043240/229278133-fc0d74d4-98db-4af0-b436-780585a5b3b6.png)

This is really nice, by the way! I think this is a new capability to remove an unmanaged layer directly from the flow editor. The flow and this capability are definitely worth a mention in Learn. Don't have to go to Command Center to remove the layer. This brings me to another point. The flow is not visible in the admin CS :)

![image](https://user-images.githubusercontent.com/64043240/229278212-9db763e5-d64d-4f5e-9d68-9cd94e520f67.png)

**Comment by manuelap-msft** [comment]
We're still in experimental preview with this feature, so appreciate your patience whilst we get our documentation etc up to date!

Now that the feature is in public preview, we've made sure setup works primarily through the setup wizard with as little as possible manual intervention required (only to configure Dataflows, which we can't automate). We're still working on documenting the manual setup instructions for the BYODL feature which would call out the correct value for the environment variables and how to get them etc, but for the preview we have prioritized getting the setup wizard working over the manual steps so there's definitely some gaps in our documentation still.
